hadoop-common-project/hadoop-common/src/main/resources/core-default.xml

```diff
@@ -1213,8 +1213,12 @@
 
 <property>
   <name>fs.s3a.connection.maximum</name>
-  <value>15</value>
-  <description>Controls the maximum number of simultaneous connections to S3.</description>
+  <value>48</value>
```

Comment:
Given the requirement mentioned below shouldn't the default fs.s3a.connection.maximum be > the default fs.s3a.threads.max?

Reply:
moved to 72. Now, 64 threads *seems* a lot, but most of these are blocking operations rather than IO heavy actions.